packages/github/src/main/ts/crawler.ts

```diff
@@ -34,16 +35,17 @@ export const createGithubCrawler = (
   }
 
   const getReposByOrgs = async (orgs: string[]) => {
-    const repos = await Promise.all(
-      orgs.map((org) =>
+    const repos = await PromisePool.all(
+      orgs,
+      (org) =>
         octokit
           .paginate(octokit.repos.listForOrg, { org })
           .then((data: any) => ({
             data,
             org,
           }))
           .catch((e: any) => e),
-      ),
+      crawlerOpts.poolSize
```

Comment:
Is this needed for octokit? afair it has own internal throttling.

Reply:
Yes, it does, but we also should limit number of promises that can exist at the moment